core/src/main/java/com/dtolabs/rundeck/core/schedule/SchedulesManager.java

```diff
@@ -99,6 +99,17 @@ public interface SchedulesManager {
      */
     List getSchedulesJobToClaim(String toServerUUID, String fromServerUUID, boolean selectAll, String projectFilter, List<String> jobids);
 
+
+    /**
+     * Gets a list of jobs that has a schedule for execution later than now
+     * @param toServerUUID
+     * @param fromServerUUID
+     * @param selectAll
+     * @param projectFilter
+     * @return List
+     */
+    List getSchedulesExecutionLater(String toServerUUID, String fromServerUUID, boolean selectAll, String projectFilter);
```

Comment:
i dont see a reason for this method to have that many params, it looks like the only one needed is the first one "toServerUUID"